assets/sass/modules/_content.scss

```diff
@@ -71,7 +71,9 @@
 	} // a
 } // .edit-link
 
+//-----------------------------------------
 // Asides
+//-----------------------------------------
 .blog .format-aside .entry-title,
```

Comment:
I know this is already in the theme, but I think it might be best to remove this and instead add the `screen-reader-text` class and ARIA labels to the entry titles of posts where we want to hide the title. Right now, the post title is hidden so on an archive page I don't have any context about the post when the screen reader gets to it.

But maybe that's intentional? I'm not sure why these were hidden in the first place. 🤔

Reply:
That's a good question. I'm going to ignore this comment in this branch and open a new issue #429